.circleci/config.yml

```
@@ -44,6 +44,39 @@ aliases:
     working_directory: ~/repo
 
 jobs:
+  build-test-no-cache:
+    <<: *defaults
+    environment:
+      JEST_JUNIT_OUTPUT_DIR: test-results
+      NODE_OPTIONS: --max-old-space-size=4000
+    steps:
+      - *restore_git_cache
+      - checkout
+      - run: npm ci
+      - run:
+          name: Lint
+          command: npm run test:lint -- --quiet --output-file test-results/eslint-results.xml --format junit
+      - run:
+          name: Unit
+          environment:
+            JEST_JUNIT_OUTPUT_NAME: unit-results.xml
+          command: npm run test:unit -- --reporters="default" --reporters="jest-junit" --coverage --coverageReporters=text --coverageReporters=lcov --maxWorkers="2"
+      - run:
+          name: Build
+          environment:
+            NODE_ENV: production
+          command: npm run build
+      - run:
+          name: Integration
+          environment:
+              JEST_JUNIT_OUTPUT_NAME: integration-results
+          command: |
+            export TESTFILES=$(circleci tests glob "test/integration/*.test.js" | circleci tests split --split-by=timings)
+            $(npm bin)/jest ${TESTFILES} --reporters="default" --reporters="jest-junit" --runInBand
```

Since we can't parallelize the single job, this can be simplified
```suggestion
            $(npm bin)/jest test/integration/*.test.js --reporters="default" --reporters="jest-junit" --runInBand
```